src/tests/ftest/launch.py

```diff
@@ -250,6 +252,8 @@ def get_output(cmd, check=True):
 
     """
     print("Running {}".format(" ".join(cmd)))
+    print("Running in cwd(): {}".format(os.getcwd()))
```

Comment:
I'm all for useful logging, but I question the usefulness of this.  Won't this always print the same directory?  As far as I know we're not changing directories within the execution of this script.  Was this possibly a debug statement that was left in?

Reply:
It was coming in handy while we were straightening out the (re)location of orterun and cart.log files. 
 Trace removed: be461626fe70c098662740a61734b597a9f7fbcb.